test/lib/runtime/panel_survey_test.exs

```diff
@@ -0,0 +1,217 @@
+defmodule Ask.Runtime.PanelSurveyTest do
+  use Ask.ModelCase
+  use Ask.TestHelpers
+  alias Ask.Runtime.{PanelSurvey, RespondentGroupAction}
+  alias Ask.{Survey, Repo, TestChannel, Respondent}
+
+  describe "new occurence" do
+    test "creates a new ready occurrence" do
+      panel_survey = completed_panel_survey()
```

Comment:
I would rename this function. The panel survey is not completed. The last ocurrence of it is completed. In any case, it would be `repeatable_panel_survey` or something like that

Reply:
Yes, I didn't like this name, but I could find a better one so I didn't change it. I'm not sure if `repeatable` is the best choice. It could be! Let's discuss it today at the call 🙏 . In the meantime, I'll change it using more descriptive (but longer) phrases.